tests/funcs/fs.sh

```diff
@@ -101,7 +101,7 @@ t_server_nr()
 t_first_client_nr()
 {
 	for i in $(t_fs_nrs); do
-		if [ "$(cat $(t_sysfs_path $i)/quorum/is_leader)" == "0" ]; then
+		if if [[ -f "$(t_sysfs_path $i)/quorum/is_leader" ]] && [[ "$(cat $(t_sysfs_path $i)/quorum/is_leader)" == "0" ]]; then
```

Comment:
`if if`? :)

This doesn't remove the race, it just makes it tighter.  The file can be removed after we test it but before we try and cat it.   The fix is to just add `2>&1` to the cats so that it compares the error message to 1 and fails rather than it ending up in the output and making the test fail.  (which is what I assume is happening :)).